<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-TV5SK76" failures="3" tests="26" name="Google featue" time="201.044" errors="0" timestamp="2023-10-29T14:54:37 IST">
  <testcase classname="com.test.ExceptionTimeoutTest" name="infinitlooptest" time="2.09">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.test.ExceptionTimeoutTest.infinitlooptest() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.test.ExceptionTimeoutTest.infinitlooptest() didn't finish within the time-out 2000
at java.base@17.0.4.1/java.io.FileOutputStream.writeBytes(Native Method)
at java.base@17.0.4.1/java.io.FileOutputStream.write(FileOutputStream.java:349)
at java.base@17.0.4.1/java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:81)
at java.base@17.0.4.1/java.io.BufferedOutputStream.flush(BufferedOutputStream.java:142)
at java.base@17.0.4.1/java.io.PrintStream.write(PrintStream.java:570)
at java.base@17.0.4.1/sun.nio.cs.StreamEncoder.writeBytes(StreamEncoder.java:234)
at java.base@17.0.4.1/sun.nio.cs.StreamEncoder.implFlushBuffer(StreamEncoder.java:313)
at java.base@17.0.4.1/sun.nio.cs.StreamEncoder.flushBuffer(StreamEncoder.java:111)
at java.base@17.0.4.1/java.io.OutputStreamWriter.flushBuffer(OutputStreamWriter.java:178)
at java.base@17.0.4.1/java.io.PrintStream.writeln(PrintStream.java:723)
at java.base@17.0.4.1/java.io.PrintStream.println(PrintStream.java:938)
at app//com.test.ExceptionTimeoutTest.infinitlooptest(ExceptionTimeoutTest.java:13)
at java.base@17.0.4.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.4.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.4.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.4.1/java.lang.Thread.run(Thread.java:833)
... Removed 8 stack frames]]>
    </failure>
  </testcase> <!-- infinitlooptest -->
  <testcase classname="com.test.ExceptionTimeoutTest" name="time" time="0.006"/>
  <testcase classname="com.test.ExceptionTimeoutTest" name="time1" time="0.233">
    <failure type="org.testng.TestException" message="
Method ExceptionTimeoutTest.time1()[pri:0, instance:com.test.ExceptionTimeoutTest@4145bad8] should have thrown an exception of type class java.lang.NumberFormatException">
      <![CDATA[org.testng.TestException: 
Method ExceptionTimeoutTest.time1()[pri:0, instance:com.test.ExceptionTimeoutTest@4145bad8] should have thrown an exception of type class java.lang.NumberFormatException
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- time1 -->
  <testcase classname="com.test.Googletest" name="t1" time="0.044"/>
  <testcase classname="com.test.Googletest" name="te3" time="0.004"/>
  <testcase classname="com.test.Googletest" name="tt2" time="0.007"/>
  <testcase classname="com.test.Googletest" name="verifytitle" time="2.28"/>
  <testcase classname="com.test.Googletest" name="verifysignbutton" time="0.38"/>
  <testcase classname="com.test.Googletest" name="verifylogo" time="0.043"/>
  <testcase classname="com.test.GoogletitileTest" name="verifylogo" time="0.155"/>
  <testcase classname="com.test.GoogletitileTest" name="verifytitle" time="0.011"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.002"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.005"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.017"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.001"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.006"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.003"/>
  <testcase classname="com.test.InvocationCount" name="sum" time="0.008"/>
  <testcase classname="com.test.TestNGbasic" name="testmethod1" time="0.004"/>
  <testcase classname="com.test.TestNGbasic" name="testmethod2" time="0.002"/>
  <testcase classname="com.test.TestNGbasic" name="testmethod3" time="0.003"/>
  <testcase classname="com.test.TestNGfeature" name="logintest" time="0.005">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at com.test.TestNGfeature.logintest(TestNGfeature.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- logintest -->
  <testcase classname="com.test.TestNGfeature" name="hometest" time="0.0">
    <skipped/>
  </testcase> <!-- hometest -->
</testsuite> <!-- Google featue -->
